lib/audited/auditor.rb

```diff
@@ -117,8 +117,11 @@ def revisions(from_version = 1)
       end
 
       # Get a specific revision specified by the version number, or +:previous+
+      # Returns nil for versions greater than revisions count
       def revision(version)
-        revision_with Audited.audit_class.reconstruct_attributes(audits_to(version))
+        if version == :previous || revisions.count >= version
```

Comment:
This will cause all revisions to be loaded from the database just so they can be counted, which can be very slow when an object had many changes.
How about comparing the version to `self.audits.last.version`?

Reply:
@tbrisker That's a great idea, I will make that change.

Reply:
Thanks @JonathanWThom ! While you're at it, can you please also add a line to the Changelog with the change?